ssa/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/fluxcd/pkg/ssa
+
+go 1.16
+
+require (
+	github.com/google/go-cmp v0.5.6
+	k8s.io/apimachinery v0.22.2
+	sigs.k8s.io/cli-utils v0.25.1-0.20210608181808-f3974341173a
```

Comment:
`cli-utils` has a loose release cycle right (and therefore pinned to a commit)?

Reply:
Yes this needs to be pinned everywhere otherwise we can't build flux due to kustomize v2.

Reply:
Would maybe add a comment (super sensitive about (documenting) version pins at the moment after libgit2 fiasco).

Reply:
Added